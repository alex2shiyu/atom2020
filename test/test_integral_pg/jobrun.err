/share/home/pengsy/.bashrc: line 3: warning: setlocale: LC_CTYPE: cannot change locale (en_US.UTF-8): No such file or directory
/share/home/pengsy/.bashrc: line 3: warning: setlocale: LC_CTYPE: cannot change locale (en_US.UTF-8)
which: no manpath in (/share/home/pengsy/soft/intel-install/compilers_and_libraries_2019.5.281/linux/mpi/intel64/bin/:/share/home/pengsy/soft/intel-install/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/opt/ibutils/bin)
/share/home/pengsy/.bashrc: line 3: warning: setlocale: LC_CTYPE: cannot change locale (en_US.UTF-8): No such file or directory
/share/home/pengsy/.bashrc: line 3: warning: setlocale: LC_CTYPE: cannot change locale (en_US.UTF-8)
Python 3.6.9 :: Anaconda, Inc.
/share/home/pengsy/work/atom2020/pg/PG_util.py:18: RuntimeWarning: invalid value encountered in true_divide
  tmp_list.append(iba/np.linalg.norm(iba))
