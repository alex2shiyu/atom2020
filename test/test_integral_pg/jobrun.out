Job starts at Sat May 23 22:53:47 CST 2020
Job ends at Sat May 23 22:53:49 CST 2020
